packages/base/src/panelview/formbuilder.tsx

```diff
@@ -56,8 +58,10 @@ export const LuminoSchemaForm = (
 export class ObjectPropertiesForm extends React.Component<IProps, IStates> {
   constructor(props: IProps) {
     super(props);
+    const sourceData = { ...this.props.sourceData };
+    this.processSourceData(sourceData);
```

Comment:
Maybe the `processSourceData()` should be executed in the constructor of `LayerPropertiesForm`, except if it can be useful for other form.